mne/simulation/evoked.py

```diff
@@ -34,9 +36,10 @@ def simulate_evoked(fwd, stc, info, cov, snr=3., tmin=None, tmax=None,
         Measurement info to generate the evoked.
     cov : Covariance object
         The noise covariance.
-    snr : float
-        signal to noise ratio in dB. It corresponds to
-        10 * log10( var(signal) / var(noise) ).
```

Comment:
It seems like having an SNR-based computation can still be useful. So we could keep both, and only allow one to be set.

But if we really want to get rid of it, we should explicitly provide the line(s) necessary to take an old SNR value and get the equivalent `nave` value to pass and put them in the `Notes`. It's actually useful pedagogically anyway because we can add that `SNR = ...` and so if you want a desired SNR, you pass `nave = ...`.